### ISSUE TITLE
files attribute must be specified

### ISSUE THREAD

**Issue by fisknils** [apifytech/apify-cli#30]
Hey, I just installed apify-cli using **npm -g i apify-cli**
I'm running Windows 7 64bit
> node -v v9.5.0 
> npm -v 6.0.0

Whenever I run apify, I get the following error messsage:

> (node:9672) Error Plugin: apify-cli: files attribute must be specified in C:\Users\fisknils\AppData\Roaming\npm\node_modules\apify-cli\package.json

I figured it might not be optimized for windows, so I also tried it on a linux box running 
> node -v v10.1.0 
> npm -v 6.1.0

But I get the (basically) the same error.

I also tried downgrading, through nvm, to the same version as someone from your support staff, v9.3.0, but sadly, this didn't solve my problems either. 
```
fisknils@linuxburk11:~$ apify
(node:28307) Error Plugin: apify-cli: files attribute must be specified in 
/home/fisknils/.nvm/versions/node/v9.3.0/lib/node_modules/apify-cli/package.json
module: @oclif/config@1.6.21
plugin: apify-cli
root: /home/fisknils/.nvm/versions/node/v9.3.0/lib/node_modules/apify-cli
See more details with DEBUG=*
Apify command line client to help you create, develop, build and run Actor acts.

VERSION
  apify-cli/0.1.9 linux-x64 node-v9.3.0

USAGE
  $ apify [COMMAND]

COMMANDS
  call    Runs the act remotely on the Apify platform.
  create  Creates a new act project directory from a selected template.
  info    Displays information about current Apify settings.
  init    Initializes an act project in an existing directory.
  login   Logs in to the Apify platform using the API token.
  logout  Logs out of the Apify platform.
  push    Uploads the act to the Apify platform and builds it there.
  run     Runs the act locally in the current directory.

```

I tried googling, but couldn't find any references to this problem. 

I'm attaching log files from running with DEBUG=*
[linux-debuglog.txt](https://github.com/apifytech/apify-cli/files/2048293/linux-debuglog.txt)
[win-debuglog.txt](https://github.com/apifytech/apify-cli/files/2048294/win-debuglog.txt)

Also, this is the output from tree on the linux box after running **apify init** in an empty directory
[tree.txt](https://github.com/apifytech/apify-cli/files/2048313/tree.txt)

The tree structure on the windows machine looks the same.

**Comment by fisknils** [comment]
Seems to be related to this:
https://github.com/oclif/config/compare/v1.6.19...v1.6.20

**Comment by drobnikj** [comment]
Yeah probably, thanks for investigation. But anyway apify-cli should work but prints this error on each command run. We fix it ASAP.

**Comment by drobnikj** [comment]
Fixed in v0.1.11
